packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -161,15 +158,18 @@ interface AppPubSubConfig {
 
 type PubSubConfig = LaunchpadPubSubConfig | AppPubSubConfig
 
+// We need a dedicated socket.io namespace, rather than re-use the one provided via the runner
+// at window.ws, because the event-manager calls .off() on its socket instance when Cypress
+// execution is stopped. We need to make sure the events here are long-lived.
```

Comment:
What's the reason we need to call `ws.off()`? I see it's a relatively recent change to `event-manager`.

Reply:
It's been around for [a lot longer](https://github.com/cypress-io/cypress/blame/676fc97f46f9798b5f17ee6d292924f8e1a0c82b/packages/runner-shared/src/event-manager.js#L626) - the recent changes were some code shuffling around:

<img width="882" alt="image" src="https://user-images.githubusercontent.com/154748/165383328-8cee9d90-4a50-4538-866f-678b06c288a4.png">

Reply:
Ah gotcha, that's what I deserve for trusting the first blame :see_no_evil: Still, do you know why?

Reply:
I guess we don't want to react to any WS events after the test is stopped 🤔, didn't dig into it